assets/css/style.css

```diff
@@ -0,0 +1,663 @@
+  align-items: center;
+}
+
+.footer .decoration {
+  position: absolute;
+  right: 0;
+  top: 0;
+}
+
+.footer .title {
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  text-align: center;
+  color: white;
+  margin-bottom: 12px;
+}
+
+.footer .desc {
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: rgb(235, 235, 255);
+  margin-bottom: 20px;
+}
+
+input,
+textarea {
+  padding: 15px;
+  margin: 9px 0;
+  background: rgb(255, 255, 255);
+  border: 1px solid rgb(207, 216, 220);
+  border-radius: 8px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+  color: rgb(23, 43, 77);
+}
+
+::placeholder {
+  color: rgb(179, 186, 197);
+}
+
+.footer .contact-button {
+  padding-top: 12px;
+}
+
+.about .info .button-outline {
+  margin-top: 0.5rem;
+  margin-bottom: 1.5rem;
+}
+
+@media all and (min-width: 375px) {
+  .is_mobile {
+    display: block;
+  }
+
+  .is_desktop {
+    display: none;
+  }
+
+  .menu {
+    justify-content: center;
+  }
+
+  .logo {
+    flex: 1;
+  }
+
+  .mobile-menu {
+    flex: 1;
+    text-align: right;
+    order: 2;
+  }
+
+  .heading::before {
+    background-image: url("../images/shape_mobile.svg");
+  }
+
+  .heading {
+    padding: 10rem 1.5rem;
+  }
+
+  nav {
+    padding-right: 15px;
+    padding-left: 10px;
+  }
+
+  .heading__title {
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  .heading__desc {
+    font-size: 16px;
+    line-height: 24px;
+  }
+
+  .tonic {
+    margin: 2rem;
+    padding: 1.5rem;
+    grid-template-columns: none;
+  }
+
+  .tonic img {
+    width: 100%;
+  }
+
+  .content {
+    padding: 24px 0;
+  }
+
+  .content .title {
+    font-size: 32px;
+    line-height: 44px;
+  }
+
+  .content .desc {
+    font-size: 15px;
+    line-height: 24px;
+  }
+
+  .content .summary {
+    font-size: 13px;
+    line-height: 16px;
+  }
+
+  .about {
+    padding: 3rem 3rem 15rem 3rem;
+    grid-template-columns: none;
+  }
+
+  .about .info,
+  .about .casualose {
+    width: 100%;
+  }
+
+  .about .casualose {
+    padding: 0;
+  }
+
+  .footer {
+    padding: 102px 12px 48px 24px;
+  }
+
+  .footer .desc {
+    width: auto;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    padding-bottom: 32px;
+  }
+
+  .footer form {
+    padding: 0 3rem 0 0;
+  }
+
+  .footer .contact-button {
+    flex-direction: row;
+    width: 100%;
+    padding-left: 0.5rem;
+  }
+
+  .footer .button-outline {
+    justify-content: flex-start;
+    flex-flow: column;
+  }
+
+  input,
+  textarea {
+    width: 100%;
+  }
+}
+
+@media all and (min-width: 768px) {
+  .menu {
+    align-items: flex-start;
+    flex-wrap: nowrap;
+  }
+
+  .logo {
+    order: 0;
+  }
+
+  .item {
+    order: 1;
+    position: relative;
+    display: block;
+    width: auto;
+    padding-top: 10px;
+    padding-bottom: 10px;
+  }
+
+  .mobile-menu {
+    display: none;
+  }
+}
+
+@media all and (min-width: 1024px) {
```

Comment:
- **[OPTIONAL]** There are some differences between your design and the Figma design. As requested by the previous CR it would have been enough to mark one breakpoint at width 768px for the desktop version 👉🏽 **@media screen and (min-width: 750px)** It will save you time to have two different styles one for the mobile version and the other for versions higher than 768px width.